app/assets/stylesheets/application.scss

```diff
@@ -104,6 +104,45 @@ main {
     dd, dt {
       margin-bottom: 3px;
     }
+
+    .in-manual-search {
+      position: relative;
+      margin: $gutter 0 $gutter-half;
+
+      label {
+        padding-bottom: 5px;
+        display: inline-block;
+      }
+
+      #search-box {
+        width: 86%;
+        width: calc(100% - 47px);
+        border: none;
+        padding: 6px;
+      }
+
+      button {
+        width: 33px;
+        height: 33px;
+        border: 1px solid #0b0c0c;
+        border-width: 0 0 0 1px;
+        border-left-color: #222;
+        -webkit-border-radius: 0;
+        -moz-border-radius: 0;
+        border-radius: 0;
```

Comment:
Could you use the mixin for this please to keep the vendor prefixing in one place: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_css3.scss#L11-L15